engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -170,8 +170,12 @@ object WorkflowExecutionActor {
         case _ => Nil
       }
 
+      def isDone(e: JobKey): Boolean = executionStore.store exists {
+        case (k, s) => k.scope.fullyQualifiedName == e.scope.fullyQualifiedName && k.index == e.index && s == ExecutionStatus.Done
```

Comment:
ToL this might benefit from a comment as to why the `exists` is required so I don't go in here and break it again in a couple of months.  😄